yarn.lock

```diff
@@ -3448,22 +3412,20 @@
     "@activeviam/icons" "5.0.15"
     "@activeviam/theme" "5.0.15"
     "@emotion/core" "^10.0.27"
-    antd "4.12.2"
```

Comment:
⚠️ This was a manual change: in this package antd is mistakenly a dependency (see https://github.com/activeviam/activeui/blob/205ad3f27e23a08c6e6c7cc647fd9b0e53e90587/packages/plugin-widget-text-editor/package.json#L27) instead of being a peer dependency.